portfolio/src/main/webapp/script.js

```diff
@@ -35,10 +35,71 @@ function addRandomQuote() {
   quoteContainer.innerText = quote;
 }
 
+/**
+ * fetch for comment section  
+ */
 function getMessageUsingArrowFunctions() {
-  fetch('/data').then(response => response.text()).then((message) => {
-    document.getElementById('message-container').innerText = message;
+  fetch('/data').then(response => response.json()).then((comment) => {
+    const commentEntry = document.getElementById('comment-container');
+    commentEntry.innerHTML = '';
+    for (let i in comment){
+      commentEntry.appendChild(createListElement(comment[i].name + " commented " + comment[i].message)); // name then message 
+    }
   });
 }
 
+/** Creates an <li> element containing text. */
+function createListElement(text) {
+  const liElement = document.createElement('li');
+  liElement.innerText = text;
+  return liElement;
+}
+
+/** Create a map */
+function createMap() {
+  // center at Taiwan presidential office building 
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 25.040, lng: 121.512}, zoom: 12
+      });
+  
+  // markers for my fav Taipei spots 
+  // location#1 Elephant Mountain Hiking Trail
+  const spotOne = new google.maps.Marker({
```

Comment:
Instead of spotOne, you can just name it ElephantMountainSpot.
Same as spotTwo and spotThree